apps/wallet/src/ui/app/pages/home/tokens/TokensDetails.tsx

```diff
@@ -101,6 +106,15 @@ export function TokenDetails({ coinType }: TokenDetailsProps) {
         refetchInterval: DELEGATED_STAKES_QUERY_REFETCH_INTERVAL,
     });
 
+    const { data: supplyIncreaseVestingObjects } = useGetAllOwnedObjects(
```

Comment:
We can add the feature of `SupplyIncreaseVesting` as in the dashboard before this